File: client/iih_stub/iih.ci_stub/iih.ci.mr/nu/obstetrics/antennurbserrec/d/AntenNurBserRecDO.cs

using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;
using System.Collections.Generic;

namespace iih.ci.mr.nu.obstetrics.antennurbserrec.d
{
    /// <summary>
    /// AntenNurBserRecDO 的摘要说明。
    /// </summary>
    public class AntenNurBserRecDO : BaseDO {

        public AntenNurBserRecDO() {
        }
		public string Id {
            get { return getAttrVal<string>("Id",null); }
            set { setAttrVal<string>("Id", value); }
        }
		public string Id_dept {
            get { return getAttrVal<string>("Id_dept",null); }
            set { setAttrVal<string>("Id_dept", value); }
        }
		public string Id_pat {
            get { return getAttrVal<string>("Id_pat",null); }
            set { setAttrVal<string>("Id_pat", value); }
        }
		public string Id_grp {
            get { return getAttrVal<string>("Id_grp",null); }
            set { setAttrVal<string>("Id_grp", value); }
        }
		public string Id_org {
            get { return getAttrVal<string>("Id_org",null); }
            set { setAttrVal<string>("Id_org", value); }
        }
		public string Ks {
            get { return getAttrVal<string>("Ks",null); }
            set { setAttrVal<string>("Ks", value); }
        }
		public string Hz {
            get { return getAttrVal<string>("Hz",null); }
            set { setAttrVal<string>("Hz", value); }
        }
		public string Ch {
            get { return getAttrVal<string>("Ch",null); }
            set { setAttrVal<string>("Ch", value); }
        }
		public string Zyh {
            get { return getAttrVal<string>("Zyh",null); }
            set { setAttrVal<string>("Zyh", value); }
        }
		public string Hznl {
            get { return getAttrVal<string>("Hznl",null); }
            set { setAttrVal<string>("Hznl", value); }
        }
		public string Rlmc {
            get { return getAttrVal<string>("Rlmc",null); }
            set { setAttrVal<string>("Rlmc", value); }
        }
		public string Rlyf {
            get { return getAttrVal<string>("Rlyf",null); }
            set { setAttrVal<string>("Rlyf", value); }
        }
		public string Bqgc {
            get { return getAttrVal<string>("Bqgc",null); }
            set { setAttrVal<string>("Bqgc", value); }
        }
		public string Clcs {
            get { return getAttrVal<string>("Clcs",null); }
            set { setAttrVal<string>("Clcs", value); }
        }
		public string Qm {
            get { return getAttrVal<string>("Qm",null); }
            set { setAttrVal<string>("Qm", value); }
        }
		public int? Mbcf {
            get { return getAttrVal<int?>("Mbcf",null); }
            set { setAttrVal<int?>("Mbcf", value); }
        }
		public int? Hxcf {
            get { return getAttrVal<int?>("Hxcf",null); }
            set { setAttrVal<int?>("Hxcf", value); }
        }
		public int? Ssymmhg {
            get { return getAttrVal<int?>("Ssymmhg",null); }
            set { setAttrVal<int?>("Ssymmhg", value); }
        }
		public int? Szymmhg {
            get { return getAttrVal<int?>("Szymmhg",null); }
            set { setAttrVal<int?>("Szymmhg", value); }
        }
		public int? Rljlml {
            get { return getAttrVal<int?>("Rljlml",null); }
            set { setAttrVal<int?>("Rljlml", value); }
        }
		public int? Gdgd {
            get { return getAttrVal<int?>("Gdgd",null); }
            set { setAttrVal<int?>("Gdgd", value); }
        }
		public int? Ydcxml {
            get { return getAttrVal<int?>("Ydcxml",null); }
            set { setAttrVal<int?>("Ydcxml", value); }
        }
		public int? Nlml {
            get { return getAttrVal<int?>("Nlml",null); }
            set { setAttrVal<int?>("Nlml", value); }
        }
		public FDouble Tw {
            get { return getAttrVal<FDouble>("Tw",null); }
            set { setAttrVal<FDouble>("Tw", value); }
        }
		public FDouble Xybhd {
            get { return getAttrVal<FDouble>("Xybhd",null); }
            set { setAttrVal<FDouble>("Xybhd", value); }
        }
		public DateTime? Jlrqsj {
            get { return getAttrVal<FDateTime>("Jlrqsj",null); }
            set { setAttrVal<FDateTime>("Jlrqsj", value); }
        }
		public string Eu_pfqk {
            get { return getAttrVal<string>("Eu_pfqk",null); }
            set { setAttrVal<string>("Eu_pfqk", value); }
        }
		public string Eu_zgss {
            get { return getAttrVal<string>("Eu_zgss",null); }
            set { setAttrVal<string>("Eu_zgss", value); }
        }
		public string Createdby {
            get { return getAttrVal<string>("Createdby",null); }
            set { setAttrVal<string>("Createdby", value); }
        }
		public DateTime? Createdtime {
            get { return getAttrVal<FDateTime>("Createdtime",null); }
            set { setAttrVal<FDateTime>("Createdtime", value); }
        }
		public string Modifiedby {
            get { return getAttrVal<string>("Modifiedby",null); }
            set { setAttrVal<string>("Modifiedby", value); }
        }
		public DateTime? Modifiedtime {
            get { return getAttrVal<FDateTime>("Modifiedtime",null); }
            set { setAttrVal<FDateTime>("Modifiedtime", value); }
        }
		public string Name_dept {
            get { return getAttrVal<string>("Name_dept",null); }
            set { setAttrVal<string>("Name_dept", value); }
        }
		public string Name_pat {
            get { return getAttrVal<string>("Name_pat",null); }
            set { setAttrVal<string>("Name_pat", value); }
        }
		public string Name_group {
            get { return getAttrVal<string>("Name_group",null); }
            set { setAttrVal<string>("Name_group", value); }
        }
		public string Name_org {
            get { return getAttrVal<string>("Name_org",null); }
            set { setAttrVal<string>("Name_org", value); }
        }
        public int Ds {
            get { return getAttrVal<int>("Ds",0);}
            set { setAttrVal<int>("Ds", value); }
        }

        public DateTime? Sv        {
            get { return getAttrVal<FDateTime>("Sv",null); }
            set { setAttrVal<FDateTime>("Sv", value); }
        }
        
        /// <summary>
        /// 返回表名
        /// </summary>
        /// <returns></returns>
        public override string getTableName()
        {
            return "CI_Mr_NU_ANTNURBSERREC";
        }
        
        /// <summary>
        /// 返回主键字段名
        /// </summary>
        /// <returns></returns>
        public override string getPKFieldName()
        {
            return "Id";
        }

        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.mr.nu.obstetrics.antennurbserrec.d.AntenNurBserRecDO";
        }

        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{"Ds","Sv",  "Id", "Id_dept", "Id_pat", "Id_grp", "Id_org", "Ks", "Hz", "Ch", "Zyh", "Hznl", "Rlmc", "Rlyf", "Bqgc", "Clcs", "Qm", "Mbcf", "Hxcf", "Ssymmhg", "Szymmhg", "Rljlml", "Gdgd", "Ydcxml", "Nlml", "Tw", "Xybhd", "Jlrqsj", "Eu_pfqk", "Eu_zgss", "Createdby", "Createdtime", "Modifiedby", "Modifiedtime", "Name_dept", "Name_pat", "Name_group", "Name_org"};
        }
        
    }
}

File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/mr/nu/obstetrics/antennurbserrec/d/AntenNurBserRecDO.java
package iih.ci.mr.nu.obstetrics.antennurbserrec.d;

import xap.sys.appfw.orm.desc.DescManager;
import xap.mw.coreitf.c.GlobalConst;
import xap.mw.coreitf.i.IDODesc;
import xap.mw.coreitf.d.*;
import xap.mw.core.data.*;
import iih.ci.mr.nu.obstetrics.antennurbserrec.d.desc.AntenNurBserRecDODesc;

/**
 * 产科护理观察记录（产后、术后） DO数据 
 * 
 */
public class AntenNurBserRecDO extends BaseDO {
	private static final long serialVersionUID = 1L;

	public static final String ID= "Id";
	public static final String ID_DEPT= "Id_dept";
	public static final String ID_PAT= "Id_pat";
	public static final String ID_GRP= "Id_grp";
	public static final String ID_ORG= "Id_org";
	public static final String KS= "Ks";
	public static final String HZ= "Hz";
	public static final String CH= "Ch";
	public static final String ZYH= "Zyh";
	public static final String HZNL= "Hznl";
	public static final String RLMC= "Rlmc";
	public static final String RLYF= "Rlyf";
	public static final String BQGC= "Bqgc";
	public static final String CLCS= "Clcs";
	public static final String QM= "Qm";
	public static final String MBCF= "Mbcf";
	public static final String HXCF= "Hxcf";
	public static final String SSYMMHG= "Ssymmhg";
	public static final String SZYMMHG= "Szymmhg";
	public static final String RLJLML= "Rljlml";
	public static final String GDGD= "Gdgd";
	public static final String YDCXML= "Ydcxml";
	public static final String NLML= "Nlml";
	public static final String TW= "Tw";
	public static final String XYBHD= "Xybhd";
	public static final String JLRQSJ= "Jlrqsj";
	public static final String EU_PFQK= "Eu_pfqk";
	public static final String EU_ZGSS= "Eu_zgss";
	public static final String CREATEDBY= "Createdby";
	public static final String CREATEDTIME= "Createdtime";
	public static final String MODIFIEDBY= "Modifiedby";
	public static final String MODIFIEDTIME= "Modifiedtime";
	public static final String NAME_DEPT= "Name_dept";
	public static final String NAME_PAT= "Name_pat";
	public static final String NAME_GROUP= "Name_group";
	public static final String NAME_ORG= "Name_org";
	public static final String DS = "Ds";
	public static final String SV = "Sv";
	
	public String getId() {
		return ((String) getAttrVal("Id"));
	}	
	public void setId(String Id) {
		setAttrVal("Id", Id);
	}
	public String getId_dept() {
		return ((String) getAttrVal("Id_dept"));
	}	
	public void setId_dept(String Id_dept) {
		setAttrVal("Id_dept", Id_dept);
	}
	public String getId_pat() {
		return ((String) getAttrVal("Id_pat"));
	}	
	public void setId_pat(String Id_pat) {
		setAttrVal("Id_pat", Id_pat);
	}
	public String getId_grp() {
		return ((String) getAttrVal("Id_grp"));
	}	
	public void setId_grp(String Id_grp) {
		setAttrVal("Id_grp", Id_grp);
	}
	public String getId_org() {
		return ((String) getAttrVal("Id_org"));
	}	
	public void setId_org(String Id_org) {
		setAttrVal("Id_org", Id_org);
	}
	public String getKs() {
		return ((String) getAttrVal("Ks"));
	}	
	public void setKs(String Ks) {
		setAttrVal("Ks", Ks);
	}
	public String getHz() {
		return ((String) getAttrVal("Hz"));
	}	
	public void setHz(String Hz) {
		setAttrVal("Hz", Hz);
	}
	public String getCh() {
		return ((String) getAttrVal("Ch"));
	}	
	public void setCh(String Ch) {
		setAttrVal("Ch", Ch);
	}
	public String getZyh() {
		return ((String) getAttrVal("Zyh"));
	}	
	public void setZyh(String Zyh) {
		setAttrVal("Zyh", Zyh);
	}
	public String getHznl() {
		return ((String) getAttrVal("Hznl"));
	}	
	public void setHznl(String Hznl) {
		setAttrVal("Hznl", Hznl);
	}
	public String getRlmc() {
		return ((String) getAttrVal("Rlmc"));
	}	
	public void setRlmc(String Rlmc) {
		setAttrVal("Rlmc", Rlmc);
	}
	public String getRlyf() {
		return ((String) getAttrVal("Rlyf"));
	}	
	public void setRlyf(String Rlyf) {
		setAttrVal("Rlyf", Rlyf);
	}
	public String getBqgc() {
		return ((String) getAttrVal("Bqgc"));
	}	
	public void setBqgc(String Bqgc) {
		setAttrVal("Bqgc", Bqgc);
	}
	public String getClcs() {
		return ((String) getAttrVal("Clcs"));
	}	
	public void setClcs(String Clcs) {
		setAttrVal("Clcs", Clcs);
	}
	public String getQm() {
		return ((String) getAttrVal("Qm"));
	}	
	public void setQm(String Qm) {
		setAttrVal("Qm", Qm);
	}
	public Integer getMbcf() {
		return ((Integer) getAttrVal("Mbcf"));
	}	
	public void setMbcf(Integer Mbcf) {
		setAttrVal("Mbcf", Mbcf);
	}
	public Integer getHxcf() {
		return ((Integer) getAttrVal("Hxcf"));
	}	
	public void setHxcf(Integer Hxcf) {
		setAttrVal("Hxcf", Hxcf);
	}
	public Integer getSsymmhg() {
		return ((Integer) getAttrVal("Ssymmhg"));
	}	
	public void setSsymmhg(Integer Ssymmhg) {
		setAttrVal("Ssymmhg", Ssymmhg);
	}
	public Integer getSzymmhg() {
		return ((Integer) getAttrVal("Szymmhg"));
	}	
	public void setSzymmhg(Integer Szymmhg) {
		setAttrVal("Szymmhg", Szymmhg);
	}
	public Integer getRljlml() {
		return ((Integer) getAttrVal("Rljlml"));
	}	
	public void setRljlml(Integer Rljlml) {
		setAttrVal("Rljlml", Rljlml);
	}
	public Integer getGdgd() {
		return ((Integer) getAttrVal("Gdgd"));
	}	
	public void setGdgd(Integer Gdgd) {
		setAttrVal("Gdgd", Gdgd);
	}
	public Integer getYdcxml() {
		return ((Integer) getAttrVal("Ydcxml"));
	}	
	public void setYdcxml(Integer Ydcxml) {
		setAttrVal("Ydcxml", Ydcxml);
	}
	public Integer getNlml() {
		return ((Integer) getAttrVal("Nlml"));
	}	
	public void setNlml(Integer Nlml) {
		setAttrVal("Nlml", Nlml);
	}
	public FDouble getTw() {
		return ((FDouble) getAttrVal("Tw"));
	}	
	public void setTw(FDouble Tw) {
		setAttrVal("Tw", Tw);
	}
	public FDouble getXybhd() {
		return ((FDouble) getAttrVal("Xybhd"));
	}	
	public void setXybhd(FDouble Xybhd) {
		setAttrVal("Xybhd", Xybhd);
	}
	public FDateTime getJlrqsj() {
		return ((FDateTime) getAttrVal("Jlrqsj"));
	}	
	public void setJlrqsj(FDateTime Jlrqsj) {
		setAttrVal("Jlrqsj", Jlrqsj);
	}
	public String getEu_pfqk() {
		return ((String) getAttrVal("Eu_pfqk"));
	}	
	public void setEu_pfqk(String Eu_pfqk) {
		setAttrVal("Eu_pfqk", Eu_pfqk);
	}
	public String getEu_zgss() {
		return ((String) getAttrVal("Eu_zgss"));
	}	
	public void setEu_zgss(String Eu_zgss) {
		setAttrVal("Eu_zgss", Eu_zgss);
	}
	public String getCreatedby() {
		return ((String) getAttrVal("Createdby"));
	}	
	public void setCreatedby(String Createdby) {
		setAttrVal("Createdby", Createdby);
	}
	public FDateTime getCreatedtime() {
		return ((FDateTime) getAttrVal("Createdtime"));
	}	
	public void setCreatedtime(FDateTime Createdtime) {
		setAttrVal("Createdtime", Createdtime);
	}
	public String getModifiedby() {
		return ((String) getAttrVal("Modifiedby"));
	}	
	public void setModifiedby(String Modifiedby) {
		setAttrVal("Modifiedby", Modifiedby);
	}
	public FDateTime getModifiedtime() {
		return ((FDateTime) getAttrVal("Modifiedtime"));
	}	
	public void setModifiedtime(FDateTime Modifiedtime) {
		setAttrVal("Modifiedtime", Modifiedtime);
	}
	public String getName_dept() {
		return ((String) getAttrVal("Name_dept"));
	}	
	public void setName_dept(String Name_dept) {
		setAttrVal("Name_dept", Name_dept);
	}
	public String getName_pat() {
		return ((String) getAttrVal("Name_pat"));
	}	
	public void setName_pat(String Name_pat) {
		setAttrVal("Name_pat", Name_pat);
	}
	public String getName_group() {
		return ((String) getAttrVal("Name_group"));
	}	
	public void setName_group(String Name_group) {
		setAttrVal("Name_group", Name_group);
	}
	public String getName_org() {
		return ((String) getAttrVal("Name_org"));
	}	
	public void setName_org(String Name_org) {
		setAttrVal("Name_org", Name_org);
	}

	public Integer getDs() {
		return ((Integer) getAttrVal(GlobalConst.DELETE_SIGN));
	}
	
	public void setDs(Integer ds) {
		setAttrVal(GlobalConst.DELETE_SIGN, ds);
	}
	
	public FDateTime getSv() {
		return ((FDateTime) getAttrVal(GlobalConst.SYSTEM_VERSION));
	}
	
	public void setSv(FDateTime sv) {
		setAttrVal(GlobalConst.SYSTEM_VERSION, sv);
	}

//	@Override
//	public java.lang.String getParentPKFieldName() {
//		return null;
//	}
  
	@Override
	public String getPKFieldName() {
		return "Id";
	}
	
	@Override
	public String getTableName() {	  
		return "CI_Mr_NU_ANTNURBSERREC";
	}
	
	@Override
	public IDODesc getDODesc() {
		return DescManager.getInstance().getDODesc(AntenNurBserRecDODesc.class);
	}
	
}